plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/__init__.py

```diff
@@ -19,9 +19,7 @@ class RayLocalLauncherConf(TargetConf):
         "hydra_plugins.hydra_ray_launcher.ray_local_launcher.RayLocalLauncher"
     )
     ray_init_cfg: Dict[str, Any] = field(default_factory=dict)
-    ray_remote_cfg: Dict[str, Any] = field(
-        default_factory=lambda: {"num_cpus": 1, "num_gpus": 0}
-    )
+    ray_remote_cfg: Dict[str, Any] = field(default_factory=dict)
```

Comment:
Do not inherit from TargetConf, it is deprecated (I am surprised that you did not see a warning.